include/ada/helpers.h

```diff
@@ -78,6 +79,10 @@ namespace ada::helpers {
    */
   ada_really_inline void strip_trailing_spaces_from_opaque_path(ada::url& url) noexcept;
 
+  /**
+   * Reverse the order of the bytes.
+   */
+  ada_really_inline uint64_t swap_bytes(uint64_t val);
```

Comment:
Can we name this to swap_bytes_if_needed and move the big endian check to here? What do you think?

Reply:
Sure. In fact, we can add a function for that and keep swap_bytes which is potentially useful on its own.

Reply:
I added swap_bytes_if_big_endian